lib/gamification_score_query.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class DiscourseGamification::GamificationScoreQuery
```

Comment:
Maybe  `DiscourseGamification::DirectoryIntegration` or something like that?

and then the method is `query` so you call `DiscourseGamification::DirectoryIntegration.query` ?